src/navigation/mobileStepper/MobileStepper.jsx

```diff
@@ -0,0 +1,184 @@
+import { Button } from 'react-cm-ui';
```

Comment:
> There are three main naming conventions in React that should be considered best practice. Components should be PascalCase – capitalized in camelCase as well and named for their function and not the specific application feature (in case you change it later).10 sep. 2020